digdag-cli/src/main/java/io/digdag/cli/client/ShowSession.java

```diff
@@ -110,17 +119,17 @@ private void showSessions(String projName, String workflowName)
 
     private void printSession(RestSession session)
```

Comment:
I think people expect that `--format json` option shows one session in json format. Or otherwise it's rejected with a message like `--format "json" is not allowed to show one session`. we can improve this in the future.